Sources/ServiceModelGenerate/ServiceModelGenerate.swift

```diff
@@ -74,21 +74,27 @@ public struct ServiceModelGenerate {
      Helper function to initialize code generation from the path to a service model.
  
      - Parameters:
-         - modelFilePath: the file path to the service model file. Supports either xml, json or yaml encoded models.
+         - modelDirectoryPath: the comma-separated paths to the service model directories. Supports either xml, json or yaml encoded models.
+         - fileExtension: the type of file extension to generate models for.
          - customizations: any customizations provided external to the model.
          - applicationDescription: the description of the application being code generated.
          - modelOverride: any overrides for values in the model.
          - generatorFunction: a function that will be provided a code generator and an instantiated ServiceModel
                               which can be used to generate any code that is required.
      */
     public static func generateFromModel<ModelType: ServiceModel>(
-        modelDirectoryPath: String, fileExtension: String,
+        modelDirectoryPath: String,
```

Comment:
Given that this is a function used by high level libraries, would it make more sense to provide an additional function override that provides an array of paths directly-

```swift
public static func generateFromModel<ModelType: ServiceModel>(        
    modelDirectoryPaths: [String],
    ...
```

This would leave it up to the higher level library to determine how this array is constructed. The existing function could then delegate to this new function with an array of one path.

Reply:
Ah yep makes sense, will make this change.